docs/supported_apis/io_supported.rst

```diff
@@ -60,3 +60,37 @@ default to pandas.
 .. _`read_sas`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_sas.html#pandas.read_sas
 .. _`read_pickle`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_pickle.html#pandas.read_pickle
 .. _`read_sql`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_sql.html#pandas.read_sql
+
+Connecting to a database for `read_sql`
+""""""""""""""""""""""""""""""""""""""""""""

```

Comment:
This line should have the same number of characters as the previous line.

Reply:
Done.